### PR TITLE
DASHBUILDE-60: Shared JSON marshalling services

### PR DESCRIPTION
**Pull request by dgutierr**

### CODE REVIEW

**Comment by manstis** [comment]
+1

**Comment by dgutierr** [comment]
This PR depends on:

https://github.com/dashbuilder/dashbuilder/pull/60

It's required due to a new module declaration "dashbuilder-json" into dashbuilder. It shall be merged together with the following dependent PRs:

https://github.com/droolsjbpm/jbpm-console-ng/pull/204
https://github.com/droolsjbpm/kie-wb-distributions/pull/125

All those PR shall be merged at the same time.

**Comment by csadilek** [comment]
Thanks.

master: 53d543e370cce089c1ba638923bd6af4de009e33

File: guvnor-webapp/pom.xml [hunk 1]
```diff
@@ -666,6 +666,7 @@
             <compileSourcesArtifact>org.kie.uberfire:kie-uberfire-social-activities-api</compileSourcesArtifact>
 
             <!-- Dashbuilder -->
+            <compileSourcesArtifact>org.dashbuilder:dashbuilder-json</compileSourcesArtifact>
```
Comment (psiroky) [inline guvnor-webapp/pom.xml]: I think we need to add `dashbuilder-json` in project dependencies as well. The build is now failing.

Comment (psiroky) [inline guvnor-webapp/pom.xml]: Ah, my bad. It seems to work even without adding to project dependencies. My failure was caused by something else.

**Pull request closed without merging**